### PR TITLE
Feature/bes graphql

### PR DESCRIPTION
**Pull request by bernd-s**
Removed the helper functions and updated readme. @andrechrist please check, if we need more examples for the usage. If this is OK, I will copy it to dev.leanix.net

### CODE REVIEW

**Comment by andrechrist** [comment]
@bernd-s Please add example for muation as well.

**Comment by bernd-s** [comment]
@rwe17, @dllx and @maikschwan  I've added the missing example. Please review the changes this morning, so that I can create a new release in the afternoon. Thank you

File: src/main/java/net/leanix/api/common/graphQL/Patch.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package net.leanix.api.common.graphQL;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Patch {
+    public enum PatchOperation {
+        add, replace, remove
```
Comment (dllx) [inline src/main/java/net/leanix/api/common/graphQL/Patch.java]: I know this is tedious, but expectations in Java are all capital constants.
So we should change this to ADD, REPLACE, ... and convert the values on serialization and deserialization.

Comment (dllx) [reply src/main/java/net/leanix/api/common/graphQL/Patch.java]: What about the other operations? (see JsonPatchOperationType in Pathfinder: there is also COPY, TEST, MOVE). Do we need them?

**Pull request merged**